core/src/navigation/ProductSwitcher.html

```diff
@@ -0,0 +1,88 @@
+{#if productSwitcherItems && productSwitcherItems.length &&
+Object.keys(productSwitcherItems[0]).length}
+<div class="fd-shellbar__action fd-shellbar__action--collapsible">
+  <div class="fd-product-switcher">
+    <div class="fd-popover fd-popover--right">
+      <div class="fd-popover__control" on:click="event.stopPropagation()">
+        <button
+          class="fd-button--shell sap-icon--grid"
+          aria-expanded="{dropDownStates.productSwitcherPopover || false}"
+          aria-haspopup="true"
+          on:click="toggleDropdownState('productSwitcherPopover')"
+        ></button>
+      </div>
+      <div
+        class="fd-popover__body fd-popover__body--right"
+        aria-hidden="{!(dropDownStates.productSwitcherPopover || false)}"
+        id="productSwitcherPopover"
+      >
+        <div class="fd-product-switcher__body">
+          <nav>
+            <ul>
+              {#each productSwitcherItems as productSwitcherItem}
+              <li on:click="onActionClick(productSwitcherItem)">
+                <span class="fd-product-switcher__product-icon">
+                  <img src="{productSwitcherItem.icon}" />
```

Comment:
We should probably add an `if` to check if an icon / label is defined. Simple change but it prevents from situations like this one: 
<img width="331" alt="screen shot 2019-01-24 at 09 07 15" src="https://user-images.githubusercontent.com/40663133/51663612-7f7ebf80-1fb7-11e9-981f-733baaf55cfb.png">